scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/VarInfoView.java

```diff
@@ -0,0 +1,205 @@
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive_grid_cloud_portal.scheduler.client.view;
+
+import java.util.Map;
+
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.Job;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.SchedulerController;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.SchedulerListeners.ExecutionDisplayModeListener;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.SchedulerListeners.JobSelectedListener;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.SchedulerListeners.JobsUpdatedListener;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.SchedulerModelImpl;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.controller.ExecutionListMode;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.model.ExecutionsModel;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.model.JobsModel;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.view.grid.ColumnsFactory;
+import org.ow2.proactive_grid_cloud_portal.scheduler.client.view.grid.KeyValueGrid;
+
+import com.smartgwt.client.types.Alignment;
+import com.smartgwt.client.widgets.Label;
+import com.smartgwt.client.widgets.layout.Layout;
+import com.smartgwt.client.widgets.layout.VStack;
+
+
+/**
+ * Displays detailed info about the currently selected job
+ *
+ *
+ * @author the activeeon team
+ *
+ */
+public class VarInfoView extends InfoView<Job>
+        implements JobSelectedListener, JobsUpdatedListener, ExecutionDisplayModeListener {
+
+    protected SchedulerController controller;
+
+    /** Generic information label text */
+    private static final String GENERIC_INFORMATION_LABEL_TEXT = "Generic Information";
+
+    /** Variables label text */
+    private static final String JOB_VARIABLES_LABEL_TEXT = "Submitted Job Variables";
+
+    /** Generic information grid */
+    private KeyValueGrid genericInformationGrid;
+
+    /** Variables grid */
+    private KeyValueGrid variablesGrid;
+
+    /**
+     * @param controller the Controller that created this View
+     */
+    public VarInfoView(SchedulerController controller, ColumnsFactory<Job> factory) {
+        super(factory, "No job selected");
```

Comment:
This line seems useless in current implementation.
You call with new VarInfoView(this.controller, null)
It doesn't cause NP exception, because of overriden build() method.
Otherwise it can be an issue. What do you think?